### ISSUE TITLE
Torrents added using RPC remain paused

### ISSUE THREAD

**Issue by wiz78** [transmission/transmission#3753]
Using the current v3 code, torrents added using https://github.com/unrelentingtech/transmitter remain in paused state, instead of starting like they did with previous versions.

transmitter doesn't specify the paused argument when calling torrent-add. AFAIK that means that Transmission should honor the start-added-torrents setting, but it doesn't.

The problem might be limited to magnet links.

**Comment by Petrprogs** [comment]
This setting is in the web interface

![Untitled](https://user-images.githubusercontent.com/76732207/188305068-9c4c6ba2-dc62-4c35-80b2-de49ae4ac582.png)

And in QT interface

![Untitled](https://user-images.githubusercontent.com/76732207/188305167-1fc8ab1b-5375-4cf5-83d6-b72875b883a7.png)

**Comment by wiz78** [comment]
yes, that one (which gets written as `"start-added-torrents": true` in settings.json

**Comment by Xist12gh** [comment]
Maybe related or same underlying issue as #3258
Didn't notice the pause problem with torrent files via transgui though, so.. maybe not?
I haven't found time to investigate myself yet.
For now I'm still basically reverting https://github.com/transmission/transmission/commit/9c0038ac8e9298b536fa8519117ebf6c0cbd289f in each build to force start remote magnets

**Comment by ckerr** [comment]
Duplicate of #3258

**Comment by wiz78** [comment]
this is not fixed by https://github.com/transmission/transmission/pull/4528 so I wouldn't say it's a duplicate of https://github.com/transmission/transmission/issues/3258

tr_torrentStartMagnet() is called only by the macOS client...

**Comment by moswain** [comment]
v4 has the same problem

**Comment by wiz78** [comment]
it's been reported again as #4715

**Comment by wborrelli** [comment]
Can confirm the same thing is happening with my new install of v4.0.0
A torrent is added, as soon as the first bits of data about file size and such is transferred, the torrent goes into a paused state. Clicking on a magnet link to add them through Tremotesf.
"start-added-torrents": true,
I have also noticed that several torrents that were added quite some time ago, and just waiting for some seeds, always go into a paused state when Transmission is restarted. I will manually start them, and then the next time Transmission is restarted, back into a paused state they will go.
